hypervisor/lazy.go

```diff
@@ -49,6 +49,13 @@ func statePreparing(ctx *VmContext, ev VmEvent) {
 		ctx.setWindowSize(cmd.ClientTag, cmd.Size)
 	case COMMAND_RUN_POD, COMMAND_REPLACE_POD:
 		glog.Info("got spec, prepare devices")
+		err := ctx.DCtx.(LazyDriverContext).InitVM(ctx)
```

Comment:
Is this better to move InitVM to LazyVmLoop? just like VmLoop call Launch to create&start vm.
And for the design, run pod only create pod,not create vm.